war/src/main/js/components/row-selection-controller/index.js

```diff
@@ -46,7 +46,7 @@ rowSelectionControllers.forEach(headerCheckbox => {
   function updateIcon() {
     headerCheckbox.classList.remove("jenkins-table__checkbox--all")
     headerCheckbox.classList.remove("jenkins-table__checkbox--indeterminate")
-    moreOptionsDropdown.classList.remove("jenkins-table__checkbox-dropdown--visible")
+    moreOptionsDropdown?.classList.remove("jenkins-table__checkbox-dropdown--visible")
```

Comment:
Likely needs a null check I don’t think this works with htmlunit

Reply:
Should be okay as we're already using optional chaining in the file 🤞